src/Register.js

```diff
@@ -31,6 +33,14 @@ export class Register extends React.Component {
         }
         validateAll(data, rules, messages)
             .then(() => {
+                this.setState({errors: {}});
+                axios.post("http://httpbin.org/post", this.state)
```

Comment:
It will work of course, but you send the whole state and errors also) If I were the server I'd prefer not to see errors I send the client earlier))

Reply:
Sorry, in `errors` only validation errors now, but anyway server doesn't need it)